br/pkg/lightning/mydump/loader_test.go

```diff
@@ -1103,3 +1105,49 @@ func TestSampleFileCompressRatio(t *testing.T) {
 	require.NoError(t, err)
 	require.InDelta(t, ratio, 5000.0/float64(bf.Len()), 1e-5)
 }
+
+func TestSampleParquetDataSize(t *testing.T) {
+	s := newTestMydumpLoaderSuite(t)
+	store, err := storage.NewLocalStorage(s.sourceDir)
+	require.NoError(t, err)
+
+	type row struct {
+		ID   int64  `parquet:"name=id, type=INT64"`
+		Name string `parquet:"name=name, type=BYTE_ARRAY, encoding=PLAIN_DICTIONARY"`
+	}
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	byteArray := make([]byte, 0, 40*1024)
+	bf := bytes.NewBuffer(byteArray)
+	pwriter, err := writer.NewParquetWriterFromWriter(bf, new(row), 4)
+	require.NoError(t, err)
+	pwriter.RowGroupSize = 128 * 1024 * 1024 //128M
+	pwriter.PageSize = 8 * 1024              //8K
+	pwriter.CompressionType = parquet.CompressionCodec_SNAPPY
+	for i := 0; i < 2000; i++ {
+		row := row{
+			ID:   int64(i),
+			Name: fmt.Sprintf("row_name_%04d", i),
```

Comment:
Can you try to generate random data and type, to test the delta is acceptable?

like use `rand` in below tests
https://github.com/pingcap/tidb/blob/89fb7adfa99932188432ad2e89d78f33f8e52874/br/pkg/lightning/backend/external/writer_test.go#L38-L40

Reply:
yesh, I used `rand` to generate test data. PTAL~